### DISCUSSION TITLE
Integrating JSBSim with Unity3d

### DISCUSSION THREAD

**Discussion by pankajchha** [JSBSim-Team/jsbsim#767] [Q&A]
Hello, I am trying to integrate JSBSim with unity3d game engine can anybody point me in right direction please ?
How to give input to JSB from unity for pitch,roll and yaw and how to utilize x,y,z rotation output of airplane from JSB in to unity

**Comment by bcoconni** [comment]
Hi @pankajchha,
Sorry for the late answer. There are similar projects that are discussed in discussions #785 and #773. May be you could investigate their approach to help you to initiate your own project ?

**Reply by pankajchha** [reply]
Hi @bcoconni Thank you very much for the replay. I will definitely check them out.
